docs/4.2/admin-guide.md

```diff
@@ -2293,6 +2294,86 @@ teleport:
 }
 ```
 
+### Using GCP Storage
+
+!!! tip "Tip":
+    Before continuing, please make sure to take a look at the
+    [cluster state section](architecture/teleport_nodes.md#cluster-state) in Teleport
+    Architecture documentation.
+
+
+GCP object storage can only be used as a storage for the recorded sessions. GCP Storage 
+cannot store the audit log or the cluster state. Below is an example of how to
+configure a Teleport auth server to store the recorded sessions in an GCP bucket.
+
+``` yaml
+teleport:
+  storage:
+      # Path to GCP Storage to store the recorded sessions in.
+      audit_sessions_uri: "sgs://Example_TELEPORT_STORAGE/records"
+      credentials_path: /var/lib/teleport/gcs_creds
```

Comment:
We should add ddvice on the credential file.
- is it just the standard GCP IAM JSON for a service account?
- what permissions does the service account require?

Reply:
I still need to figure this out, I'm building out a test cluster now to figure out the best way to do this.